f5_openstack_agent/lbaasv2/drivers/bigip/resource_manager.py

```diff
@@ -534,6 +542,15 @@ def update(self, old_listener, listener, service, **kwargs):
     @serialized('ListenerManager.delete')
     @log_helpers.log_method_call
     def delete(self, listener, service, **kwargs):
+        loadbalancer = service.get("loadbalancer", None)
```

Comment:
What is the purpose to run this piece of code before deleting a VS? The resource path of a VS does not contain route domain information. What is your expected result?

Reply:
> What is the purpose to run this piece of code before deleting a VS? The resource path of a VS does not contain route domain information. What is your expected result?

Yes, do not need this, vs is deleted by name, not whole payload.